documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -1,5 +1,12 @@
 # Current Release notes
 
+[company_name] [solution_name] has been renamed [detect_product_long] with page links, documentation, and other URLs updated accordingly. Update any [detect_product_short] documentation, or other bookmarks you may have. See the [Domain Change FAQ](https://community.blackduck.com/s/article/Black-Duck-Domain-Change-FAQ).
+* As part of this activity, sig-repo.synopsys.com and detect.synopsys.com are being deprecated. Please make use of repo.blackduck.com and detect.blackduck.com respectively. 
+    * After February 2025, [detect_product_short] script download details will only be available via detect.blackduck.com.
+    * [detect_product_short] 10.0.0 and later will only work when using repo.blackduck.com.
+
```

Comment:
@sig-cpotts Do you think we should bring in those two bullet points here from 9.10.0 and 8.11.2. If using, "it is essential....." lines?! That completes this PR! Lol

Reply:
Good idea. Done.